openc3/lib/openc3/api/tlm_api.rb

```diff
@@ -137,8 +137,11 @@ def inject_tlm(target_name, packet_name, item_hash = nil, type: :CONVERTED, scop
         end
       end
       packet.received_time = Time.now.sys
-      # TODO: New packet so received_count is not correct
-      packet.received_count += 1
+      begin
+        packet.received_count = CvtModel.get_item(target_name, packet_name, "RECEIVED_COUNT", type: :CONVERTED, scope: scope) + 1
```

Comment:
Probably should pull from the last message in the Raw topic?   That would be the most recent received_count.  There will always be a race condition with this, but the CVT doesn't get updated until after decom, so for high rate packets this will probably be behind.  (But then for high rate packets why are you using inject_tlm).  Don't know.